build.sbt

```diff
@@ -50,7 +56,7 @@ lazy val databaseSql = (project in file("database/sql"))
 lazy val databaseMigration = (project in file("database/migration"))
   .withLibrarySettings("cromwell-database-migration", databaseMigrationDependencies)
   .dependsOn(core)
-  .dependsOn(wdl)
+  .dependsOn(wdlDraft2)
```

Comment:
I'm not sure I understand what's going to happen when draft-3 starts diverging. is this expected to depend on both draft2 and 3 ?

Reply:
AFAIK this dependency is for a one-off migration which needed access to some WDL objects, as they were at the time of the migration. A nice effect of this hard split is that it allows us to freeze in time the objects which this historical migration relies on.